color_pair.cpp

```diff
@@ -0,0 +1,46 @@
+#include "color_pair.h"
+using namespace TelCoColorCoder;
+using namespace std;
+
+void TelCoColorCoder::testNumberToPair(int pairNumber,
+    TelCoColorCoder::MajorColor expectedMajor,
+    TelCoColorCoder::MinorColor expectedMinor)
+{
+    TelCoColorCoder::ColorPair colorPair =
+        TelCoColorCoder::GetColorFromPairNumber(pairNumber);
+    std::cout << "Got pair " << colorPair.ToString() << std::endl;
+    assert(colorPair.getMajor() == expectedMajor);
+    assert(colorPair.getMinor() == expectedMinor);
+}
+
+void TelCoColorCoder::testPairToNumber(
+    TelCoColorCoder::MajorColor major,
+    TelCoColorCoder::MinorColor minor,
+    int expectedPairNumber)
+{
+    int pairNumber = TelCoColorCoder::GetPairNumberFromColor(major, minor);
+    std::cout << "Got pair number " << pairNumber << std::endl;
+    assert(pairNumber == expectedPairNumber);
+}
+
+void TelCoColorCoder::printManual() 
```

Comment:
Printing of manual is together with test functionality. That means I cannot reuse this function without taking the tests as well. Separate tests and new functionality into their own files.